proxy/envoy/testdata/cb-policy.yaml.golden

```diff
@@ -1,16 +1,25 @@
 type: destination-policy
-name: world
+name: circuit-breaker
 namespace: default
+domain: cluster.local
```

Comment:
Is this honored? If so, is this the answer for https://groups.google.com/forum/#!topic/istio-users/_X3N6tZfwOc?

Reply:
Yes, it is respected to derive the FQDN for the services. It's inserted by our CRD adaptor using "domainSuffix" flags passed to the server.